packages/loader/container-loader/src/container.ts

```diff
@@ -602,10 +597,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 // This should be fired after taking the summary because it is the place where we are
                 // starting to attach the container to storage.
                 // Also, this should only be fired in detached container.
-                if (this.context.runtimeVersion !== undefined && this.context.runtimeVersion >= "0.25") {
```

Comment:
Good that we remove it. Noting that string based version comparison doesn't work,  and we should think about how to make sure this kind of bugs doesn't get into the code base